morango/api/viewsets.py

```diff
@@ -18,6 +18,7 @@ class CertificateViewSet(viewsets.ModelViewSet):
     serializer_class = serializers.CertificateSerializer
     authentication_classes = (permissions.BasicMultiArgumentAuthentication,)
 
+    # @decorators.authentication_classes(permissions.BasicMultiArgumentAuthentication,)
```

Comment:
I was wondering whether we should only have auth permissions on creating a cert. And allow anyone to query the certs as in `get_remote_certificates`, which will get all certs the server owns.